peribolos/knative-extensions.yaml

```diff
@@ -641,7 +639,6 @@ orgs:
             privacy: closed
             description: The Working Group leads for Eventing Kafka
             members:
```

Comment:
@pierDipi we should probably consolidate the eventing kafka teams - since this is empty